### ISSUE TITLE
Docs request: config

### ISSUE THREAD

**Issue by rektide** [ondreian/ebay-promised#46]
Hi hello. Great project, thanks.

I found how to use env vars to get the project up and going, which is great, but the getting going experience was rough and I'm not sure what I'm missing from not having a config object. It'd be a great help to have some docs on config, which would really ease the initial getting-started work.

Thanks for the project.

**Comment by ondreian** [comment]
Good point, this was one of the reasons I wanted to add Flow typings because it can spit out information in the documentation about the Object shape that `Ebay.create` expects.

I will at least add better documentation about this on the next release for bug fixes I make.

Here is an example of what `Ebay.create` expects:
```
Ebay.create({
        authToken : process.env.EBAY_TOKEN   || throws.Env_Error("EBAY_TOKEN")
      , cert      : process.env.EBAY_CERT    || throws.Env_Error("EBAY_CERT")
      , app       : process.env.EBAY_APP_ID  || throws.Env_Error("EBAY_APP_ID")
      , devName   : process.env.EBAY_DEV_ID  || throws.Env_Error("EBAY_DEV_ID")
      , sandbox   : process.env.EBAY_SANDBOX || false
    })
```
all of those keys (except the sandbox option) are required.

**Comment by camposfyi** [comment]
I agree, there's very little documentation on how to use this module. My server fails to start just by requiring ebay-promised without any console output.

**Comment by ondreian** [comment]
I cannot recreate this problem.  Can you please provide:

1. a minimal setup that produces this issue
2. node version you are using when this issue occurs.

recently esdoc.org seems to be having some issues, I'll see about building the docs and self-hosting for now. 

Every call described in the ebay trading xml api documentation should be available and every `ebay-promised` specific public method has JSDoc style documentation & there are clear concise examples in `test/`.
